### DISCUSSION TITLE
Connecting to TV/Console?

### DISCUSSION THREAD

**Discussion by 91dns** [hrkfdn/ncspot#1473] [Q&A]
Hello,

is there a way to link the music playing to my xbox/tv?

**Comment by Bettehem** [comment]
Not possible with the current implementation. You can find a discussion about this here: [#83](https://github.com/hrkfdn/ncspot/issues/83)

**Comment by 91dns** [comment]
Thank you for your quick answer. Can be closed.

Sent from Proton Mail Android

-------- Original Message --------
